website/client/components/yesterdailyModal.vue

```diff
@@ -7,27 +7,42 @@
     :no-close-on-esc="true",
     @hide="$emit('hide')",
   )
-    .modal-body
-      h1.header-welcome.text-center {{ $t('welcomeBack') }}
-      p.call-to-action.text-center {{ $t('checkOffYesterDailies') }}
-      .tasks-list
-        task(
-          v-for="task in tasksByType.daily",
-          :key="task.id",
-          :task="task",
-          :isUser="true",
-          :dueDate="dueDate",
-        )
-      .start-day.text-center
-        button.btn.btn-primary(@click='close()') {{ $t('yesterDailiesCallToAction') }}
+    h1.header-welcome.text-center {{ $t('welcomeBack') }}
+    p.call-to-action.text-center {{ $t('checkOffYesterDailies') }}
+    .tasks-list
+      task(
+        v-for="task in tasksByType.daily",
+        :key="task.id",
+        :task="task",
+        :isUser="true",
+        :dueDate="dueDate",
+      )
+    .start-day.text-center
+      button.btn.btn-primary(@click='close()') {{ $t('yesterDailiesCallToAction') }}
 </template>
 
+<style lang="scss">
+  #yesterdaily {
+    .modal-dialog {
+      width: 362px;
```

Comment:
@Tressley are we sure we want to use such precise pixels dimensions? In this case I think it will work well on small devices since the width is small but we risk making this very error prone due to unrelated future css changes

Reply:
@paglias -- I'd prefer we start using `rem` as our base units. If we set `font-size: 100%` on the root element, that gives us a 16px baseline to work with. In this case it'd become `width: 22.625rem;` If necessary, we could adjust the width slightly to use whole numbers instead.

Reply:
@Tressley I agree about rems (we can already use them, bootstrap's base css already sets the correct size for the root element) but I feel like using absolute sizes in a responsive layout  is a bit problematic per se, for example in this case while nothing disastrous happens if you resize the screen up to a certain size you get this:

<img width="578" alt="Schermata 2019-09-09 alle 16 06 54" src="https://user-images.githubusercontent.com/589911/64537990-208a7280-d31c-11e9-8789-c24d7053d18e.png">

A couple of pixels more and the modal gets wider:

<img width="586" alt="Schermata 2019-09-09 alle 16 07 01" src="https://user-images.githubusercontent.com/589911/64538019-2da76180-d31c-11e9-93de-c2c77028d442.png">